app/views/candidate_interface/application_choices/review.html.erb

```diff
@@ -1,13 +1,24 @@
 <% content_for :title, t('page_titles.courses') %>
 <% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
 
-<h1 class="govuk-heading-xl">
-  <% if @application_form.candidate_can_choose_single_course? %>
-    <%= t('page_titles.course_choice') %>
-  <% else %>
-    <%= t('page_titles.course_choices') %>
-  <% end %>
-</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <% if @application_form.candidate_can_choose_single_course? %>
+        <%= t('page_titles.course_choice') %>
+      <% else %>
+        <%= t('page_titles.course_choices') %>
+      <% end %>
+    </h1>
+
+    <% if @application_choices.present? && @application_form.can_add_more_choices? %>
+      <div class="govuk-inset-text">
+        <p class="govuk-body"><%= "You can choose #{current_application.choices_left_to_make} more #{'course'.pluralize(current_application.choices_left_to_make)}." %></p>
```

Comment:
I think you could use the [`pluralize`](https://api.rubyonrails.org/v6.1.0/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize) helper here to simplify this, by pluralizing "more course" / "more courses".

```suggestion
        <p class="govuk-body">You can choose <%= pluralize(current_application.choices_left_to_make), "more course" %>.</p>
```

Reply:
👌